docs/add_custom_features.md

```diff
@@ -96,13 +96,73 @@ It will return counts of each combination of values for the features named in `f
 
 where
 
-* `feature-level` represents the fine-grained level of the analysis
+* `feature-level`: the fine-grained level of the analysis
   * `example` for sentence-level analysis
   * `span` for span-level analysis (e.g. in named entity recognition analysis)
   * `token` for token-level analysis (e.g. in named entity conditional text generation)
-* `features` should be a list of feature names where each feature are predefined. These
+* `features`: a list of feature names where each feature are predefined. These
   features can be true_label, predicted_label, or other custom features.
 
+## Example of CalibrationAnalysis
+
+CalibrationAnalysis is to measure the descrepency between the predicting
+probability and the true correctness likelihood. It must come together
+with the accuracy metric. For tasks with a default accuracy metric,
+calibration analysis is automatically perform on `confidence` feature.
+You can directly add the `confidence` values to the examples in the json output.
+You may also customize calibration analysis with custom feature name and settings.
+
+Default calibration analysis format [example](../data/system_outputs/absa/absa-example-output-confidence.json).
+
+```json
+{
+  "examples": [
+    {
+      "predicted_label": "positive",
+      "confidence": 0.22101897026820283
+    }
+  ]
+}
+```
+
+Custom calibration analysis format [example](../data/system_outputs/absa/absa-example-output-custom-calibration-analysis.json).
+
+```json
+{
+  "metadata": {
+    "custom_features": {
+      "example": {
+        "log_likelihood": {
```

Comment:
This isn't the log likelihood because it's positive. Maybe you just mean `probability`?